lib/widgets/content.dart

```diff
@@ -56,6 +56,22 @@ class InheritedMessage extends InheritedWidget {
   }
 }
 
+class ThematicBreak extends StatelessWidget {
```

Comment:
This should come after `BlockContentList`, like the other widgets that are used there.

(Just in the same order as those, as mentioned at https://github.com/zulip/zulip-flutter/pull/603#discussion_r1545946957 .)